templates/zerver/integrations.html

```diff
@@ -101,128 +101,102 @@
             {% if integration.doc %}
                 <div id={{ integration.name }} class="integration-instructions">
                     {% include integration.doc %}
+                    <p style="font-size:11px; font-style:italic;">
+                        Logos are trademarks of their respective owners.
+                        None of the integrations on this page are created by,
+                        affiliated with, or supported by the companies
+                        represented by the logos.
+                    </p>
```

Comment:
Rather than copying this block in 15 places, can we put it into a template `integrations_disclaimer.html` and then just include it where it's needed?  Syntax is e.g. `{% include "zerver/left_sidebar.html" %}`